loops.js

```diff
@@ -16,9 +18,12 @@
 // loop...
 // loop...
 // klaar!
- // ==========================================
-
+// ==========================================
 
+for (let i = 0; i < 4; i++) {
+  console.log("Loop.....");
+}
+console.log("klaar!");
```

Comment:
Functioneel geeft het de juiste uitkomst weer. In de uitwerkingen van NOVI wordt echter gebruik gemaakt van een if else statement op de volgende manier:
for (let i = 0; i < 5; i++) {
if (i === 4) {
console.log('klaar!')
} else {
console.log('loop...');
}
}

Zelf heb ik ook een if else statement gebruikt maar dan weer net iets anders.